UniAtHome/UniAtHome.WebAPI/Extensions/StartupExtensions.cs

```diff
@@ -20,7 +45,11 @@ public static IServiceCollection AddIdentityConfiguration(this IServiceCollectio
             services.AddIdentity<User, IdentityRole>(
                     config =>
                     {
-                        //Please, add identity configuration here
+                        // Our UserNames = Emails. When registering new user with already registered email
+                        // there will be two unique violations unless emails ain't requiered to be unique.
+                        config.User.RequireUniqueEmail = false;
```

Comment:
I guess we should change it to "true"

Reply:
If it's true, the default error message about registering 2 users with the same email will produce 2 errors of the same kind ( 1) already exists user with this name 2) already exists user with this email). So when this property is false, there is only message #1.